bin/python/combine_reports_from_date_range.py

```diff
@@ -0,0 +1,109 @@
+"""Script for combining reports"""
+import os
+import sys
+import datetime
+import re
+import json
+import functools
+
+import logger
+import configurator
+from gcp import GcpManager
+from report import MdTable
+
+LOGGER = logger.get_logger('combine_reports_from_date_range')
+def _reduce_labels(sets):
```

Comment:
should be two blank lines between all top level elements. I'm guessing the linter isn't being run on this file...  Can you either make sure bin/check_style runs lint on this directory in this PR, or create a bug for somebody to do it later!

Reply:
I checked this (two blank lines between all top level elements) is actually not a requirement under current linter. I put these 2 lines but If this should be required, linter may need to be modified.